apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -423,10 +419,10 @@ function wpcom_display_global_styles_launch_bar( $bar_controls ) {
 
 	if ( wpcom_is_previewing_global_styles() ) {
 		$preview_text     = __( 'Turn off preview', 'full-site-editing' );
```

Comment:
A separate issue perhaps but I think this is unclear in context:

![image](https://github.com/Automattic/wp-calypso/assets/93301/919d527a-799b-42ee-b8bb-6760174ec587)

Is it clear that this preview toggle is only for customized styles?

I think it should be 'Show customized styles' and 'Hide customized styles', or maybe "View with customized styles" and "View without customized styles"